doc/contributor-how-to/how-to-dev-plugin.md

```diff
@@ -0,0 +1,64 @@
+# How to install QGreenland Custom as a developer
+
+QGreenland Custom is a QGIS plugin for downloading a custom set of data, including data which 
+is not part of the QGreenland Core zip package. This how-to guide walks through how to install the plugin as a developer.
+
+First, clone the QGreenland Custom repo from Faunalia:
+
+```
+git clone https://github.com/faunalia/qgreenland-plugin.git
+```
+
+Use the QGIS GUI to locate the QGreenland Custom directory by going to
+Settings -> User Profiles -> Open Active Profile Folder.
+Then, navigate to the /python/plugins directory, PWD and copy the full filpath. 
+
+Next symlink the plugin repo to the plugin directory:
+```
+ln -s $PWD '/filepath'
+```
+## Installing QGreenland Custom
+
+To install QGreenland Custom, navigate to plugins -> Installed -> Check box on QGreenland -> OK
+
+Instead of restarting QGIS after installing QGreenland Custom, 
+install the plugin, Plugin Reloader. You can do this by navigating to 
+Plugin -> Manage and Install Plugins, then searching for Plugin Reloader in the
+search bar. Install the plugin then press OK.
+
+To use the plugin, navigate to Plugins -> Plugin Reloader -> Configure.
+Select qgreenland-plugin in the drop down. Uncheck the 'run the commands
+below before reloading' option. Leave the 'display' option checked, and press OK.
+
+![configure_plugin](/_images/configure_plugin.png)
+
+Navigate to Plugins -> Plugin Reloader -> Reload Plugin: qgreenland-plugin.
+
+![reload_plugin](/_images/reload_plugin.png)
+
+You should receive a message notifying you that the plugin has been reloaded.
+
+## Configuring QGreenland Custom
+To use QGreenland Custom, first navigate to Web -> QGreenland -> Configure the Server. Leave the default options and press OK. 
+
+![configure_server](/_images/configure_server.png)
+
+## Using the plugin to download data
```

Comment:
All the stuff in this subsection is user documentation. Is this present elsewhere on our doc site? If so, we should remove it completely, otherwise we should move it from the "How to develop QGreenland Custom" to a new doc.

Reply:
Looking below, this seems to be covered in "Searching for data layers" and "Downloading data" in `plugin-how-to.md`. Is there any information in this section that is not in the other doc?

Reply:
I don't think so, I think I copied it directly

Reply:
OK. We should just have one copy, and it should probably be in the user how-to, not the contributor how-to!

- [ ] Ensure all documentation content is organized for the correct audience, contributor or end-user.